frontends/ast/simplify.cc

```diff
@@ -300,13 +300,37 @@ static int size_packed_struct(AstNode *snode, int base_offset)
 	bool is_union = (snode->type == AST_UNION);
 	int offset = 0;
 	int packed_width = -1;
+	// embeded struct or union with range?
+	auto it = std::remove_if(snode->children.begin(), snode->children.end(),
```

Comment:
While I think this is a good feature to add, it seems to be both out of scope and untested. Can we split this out?

Reply:
Removed from this PR. I accidentally cherry-picked this change with commit contains other changes necessary for this PR.